src/mono/netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -126,7 +126,6 @@
 
     <FeatureManagedEtwChannels>true</FeatureManagedEtwChannels>
     <FeatureManagedEtw>true</FeatureManagedEtw>
-    <FeaturePortableTimer>true</FeaturePortableTimer>
```

Comment:
This will probably need a `Condition="'$(TargetsBrowser)' != 'true'"` instead of removing since it breaks iOS/Android otherwise.